### PR TITLE
Buffs the sniper rifle

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Buffs the sniper rifle:
- Firerate decreased from 4 seconds to 0.7 seconds
- Damage decreased from 70 to 45
- Paralysis decreased from 10 seconds to 1 seconds
- Knockdown added for 4 seconds

## Why It's Good For The Game

The sniper rifle is currently a very weak weapon, due having an insane 4 seconds shot delay which makes it almost unusable, combined with a low ammo count.
This reduces the sniper rifle's damage, making it more effective at combat ranges.
It's role now serves as a slow firing, high damage, armour penetrating weapon will is very effective against alone, armoured targets.

This makes the sniper rifle incredibly powerful and a weapon that you want to be careful when approaching, rather than a weapon that is almost entirely useless.
I get that its a sniper rifle, but in SS13 combat terms some kind of DMR/marksman rifle makes more balancing sense.

## Testing Photographs and Procedure

https://user-images.githubusercontent.com/26465327/221035008-38c06549-c1ab-4553-8e10-47ea80410d91.mp4

## Changelog
:cl:
balance: Sniper rifle firerate decreased from 4 seconds to 0.7 seconds
balance: Sniper rifle damage decreased from 70 to 45
balance: Removes sniper rifle paralysis
balance: Sniper rifle knockdown added for 4 seconds
balance: Sniper rifle soporific bullets now deal 25 damage instead of 0
balance: Sniper rifle penetrator rounds now deal 40 damage
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
if you increase the fire rate that much, should really reduce or remove paralysis. it being what it was before is essentially a single target pick weapon, but like this it can take on crowds of people

File: code/modules/projectiles/projectile/bullets/sniper.dm [hunk 1]
```diff
@@ -3,8 +3,9 @@
 /obj/item/projectile/bullet/p50
 	name =".50 bullet"
 	speed = 0.4
-	damage = 70
-	paralyze = 100
+	damage = 45
+	paralyze = 10
```
Comment (PowerfulBacon) [inline code/modules/projectiles/projectile/bullets/sniper.dm]:
```suggestion
```

File: code/modules/projectiles/projectile/bullets/sniper.dm [hunk 2]
```diff
@@ -17,10 +18,11 @@
 
 /obj/item/projectile/bullet/p50/soporific
 	name =".50 soporific bullet"
-	armour_penetration = 0
-	damage = 0
+	armour_penetration = 20
+	damage = 25
 	dismemberment = 0
 	paralyze = 0
```
Comment (PowerfulBacon) [inline code/modules/projectiles/projectile/bullets/sniper.dm]:
```suggestion
```

File: code/modules/projectiles/projectile/bullets/sniper.dm [hunk 3]
```diff
@@ -32,11 +34,12 @@
 /obj/item/projectile/bullet/p50/penetrator
 	name =".50 penetrator bullet"
 	icon_state = "gauss"
-	damage = 60
+	damage = 40
 	projectile_piercing = PASSMOB
 	projectile_phasing = (ALL & (~PASSMOB))
 	dismemberment = 0 //It goes through you cleanly.
 	paralyze = 0
```
Comment (PowerfulBacon) [inline code/modules/projectiles/projectile/bullets/sniper.dm]:
```suggestion
```

**Comment by PowerfulBacon** [comment]
I agree, removed the paralysis but the knockdown remains

**Comment by Rukofamicom** [comment]
Summarizing my thoughts as discussed with you via discord

If you want to introduce a marksman rifle for use in assault scenarios, introduce a new gun instead of completely changing a weapon that has its own niche already. The sniper rifle is admittedly not well suited to team nukies unless they have some next-level organization, but it does fulfil its role extremely well for the lavaland syndicate when it comes to defending their base, and is a pretty solid choice for lone operatives who don't have to worry about team-killing due to its ability to immediately zero someone.

**Comment by PowerfulBacon** [comment]
The stupid fire delay is the main thing that kills all satisfaction of using the weapon, I don't mind keeping it with overpowered damage and just reducing the fire delay (the ammo is so expensive it kind of balances itself out with almost 1 TC per shot)

**Comment by Tsar-Salat** [comment]
> I get that its a sniper rifle, but in SS13 combat terms some kind of DMR/marksman rifle makes more balancing sense.

If you want a DMR, make a DMR. Theres numerous guns in our code that don't even see the light of day. 

Just balance those to fit the niche you are going for, theres no need to kill the niche the Sniper Rifle is in.

**Comment by benbot16** [comment]
The sniper rifle's really fine on its own, especially if you've got people backing you up or a decent secondary. It fits into a backpack, meaning you can also combine it with any number of other weapons fairly easily.
To name a few uses:
- Penetration ammo with any type of thermals/x-rays can easily take out stationary targets like AIs from up to 50 tiles away and damage anyone dumb enough to line up in a hallway.
- Soporific ammo's good in stealth situations, especially when combined with a silencer. The 40 second sleep's also good for kidnapping people if you feel like doing that.
- A rifle chambered in regular ammo gives you an easy stun and delimb if you're lucky, which you or an ally can follow up on with another weapon, such as the free stetchkin or a C20.


If you want something that functions as a DMR, then rework the surplus rifle instead (the ballistic/automatic one, not the mosin one). That things basically an M14 anyways, and it'd fit the bill for a marksman's rifle.

**Comment by PowerfulBacon** [comment]
Alright, I see the want for a weapon that can act in the niche territory of a sniper rifle, so I will present an alternative

**Pull request closed without merging**